projects/Rosenpass/services/basic/tests/basic.nix

```
@@ -0,0 +1,28 @@
+{
+  sources,
+  ...
+}:
+
+{
+  name = "Program Name";
+
+  nodes = {
+    machine =
+      { ... }:
+      {
+        imports = [
+          sources.modules.ngipkgs
+          sources.modules.programs._serviceName_
+          sources.examples._ProjectName_._exampleName_ # i.e _ProjectName_.basic
```

1. You need to replace `_serviceName_` with the actual service name...
2. But let's just delete this file since it doesn't test anything

```suggestion
          sources.modules.programs._serviceName_
          sources.examples._ProjectName_._exampleName_ # i.e _ProjectName_.basic
```